plugin/src/main/java/net/dzikoysk/funnyguilds/shared/bukkit/FunnyBox.java

```diff
@@ -311,7 +322,8 @@ public FunnyBox union(double posX, double posY, double posZ) {
         double newMaxY = Math.max(this.maxY, posY);
         double newMaxZ = Math.max(this.maxZ, posZ);
 
-        if ((newMinX == this.minX) && (newMinY == this.minY) && (newMinZ == this.minZ) && (newMaxX == this.maxX) && (newMaxY == this.maxY) && (newMaxZ == this.maxZ)) {
+        if ((newMinX == this.minX) && (newMinY == this.minY) && (newMinZ == this.minZ) && (newMaxX == this.maxX) &&
+                (newMaxY == this.maxY) && (newMaxZ == this.maxZ)) {
```

Comment:
It still isn't as readable as it could. Może tak?
```suggestion
        if ((newMinX == this.minX) && (newMinY == this.minY) &&
            (newMinZ == this.minZ) && (newMaxX == this.maxX) &&
            (newMaxY == this.maxY) && (newMaxZ == this.maxZ)) {
```

Reply:
Corrected